loadtest/control/simplecontroller/actions.go

```diff
@@ -47,6 +47,19 @@ func (c *SimpleController) sendDirectMessage(userID string) control.UserStatus {
 }
 
 func (c *SimpleController) scrollChannel(u user.User) control.UserActionResponse {
+	collapsedThreads := c.user.Store().ClientConfig()["CollapsedThreads"] == model.COLLAPSED_THREADS_DEFAULT_ON
```

Comment:
Is this needed? I was thinking that relying on the preference value would be enough.

Reply:
@streamer45 user preferences don't handle default cases. So if system is set to 'default_on' and user didn't change preferences - we need to consider the feature as 'enabled'.
So just checking preferences is not enough

Reply:
Perfect, that makes total sense then :+1: